README.md

```diff
@@ -307,3 +307,21 @@ end
 ```
 
 Each piece of stream will be encoded as json and sent as a new line (conforming to json stream format)
+
+
+## Multipart functionality
+
+You can pass a Multipart struct as the body.
+
+Example:
+```ex
```

Comment:
I think this example is missing `alias Tesla.Multipart`. It could be misleading for someone who read only a readme.